src/intTest/resources/success_uat_data/NHS004/screening_full_acceptance_with_nhsack.edifact.dat

```diff
@@ -0,0 +1,43 @@
+UNA:+.? '
+UNB+UNOC:3+000000004400001+000000024600002+091113:1519+34++SCRN_BCS_MEDRPT_V4++1'
```

Comment:
Why "SCRN_BCS_MEDRPT_V4"? Is it not usually just "MEDRPT" for both pathology and screening messages?

Reply:
Somewhere along the line I started putting in the workflows, I can't remember why I started doing that but the documentation seems to say that "MEDRPT" is used for both, so I'll change that.

Reply:
I'll fix the other edifact files as well, as they were using workflowIDs.

Reply:
Upon investigating, I've realised that fixing will require some changes to tests and to NhsAckProducerService in main, I should be able to fix them easily enough but this PR may become unfocused.

Should I make those changes here or make a separate PR?

Reply:
As discussed in teams, this issue will be fixed in a separate PR.